packages/flutter/test/material/time_picker_test.dart

```diff
@@ -134,6 +134,36 @@ void _tests() {
     expect(result.hour, equals(9));
   });
 
+  testWidgets('tap-select switches from hour to minute', (WidgetTester tester) async {
+    TimeOfDay result;
+
+    final Offset center = await startPicker(tester, (TimeOfDay time) { result = time; });
+    await tester.tapAt(Offset(center.dx, center.dy + 50.0));
```

Comment:
It would be nice to define variables for the offsets, like hour6, like in the other tests.